lib/capability/capability.go

```diff
@@ -131,28 +131,13 @@ func Contains(caps []configv1.ClusterVersionCapability, capability configv1.Clus
 	return found
 }
 
-// setKnownCapabilities populates a map keyed by capability from all known capabilities as defined in ClusterVersion.
-func setKnownCapabilities() map[configv1.ClusterVersionCapability]struct{} {
-	known := make(map[configv1.ClusterVersionCapability]struct{})
-
-	for _, v := range configv1.ClusterVersionCapabilitySets {
-		for _, capability := range v {
-			if _, ok := known[capability]; ok {
-				continue
-			}
-			known[capability] = struct{}{}
-		}
-	}
-	return known
-}
-
 // setEnabledCapabilities populates a map keyed by capability from all enabled capabilities as defined in ClusterVersion.
 // DefaultCapabilitySet is used if a baseline capability set is not defined by ClusterVersion. A check is then made to
 // ensure that no previously enabled capability is now disabled and if any such capabilities are found each is enabled,
 // saved, and returned.
 // The required capabilities are added to the implicitly enabled.
 func setEnabledCapabilities(capabilitiesSpec *configv1.ClusterVersionCapabilitiesSpec,
-	priorEnabled, alwaysEnabled map[configv1.ClusterVersionCapability]struct{}) (map[configv1.ClusterVersionCapability]struct{},
+	priorEnabled []configv1.ClusterVersionCapability) (map[configv1.ClusterVersionCapability]struct{},
```

Comment:
Same like above. We now take union of `priorEnabled` and `alwaysEnabled` which means `priorEnabled` is probably not a good name.